README.md

```diff
@@ -1,2 +1,288 @@
+  * [Execution](#execution)
+* [Theory and Approach](#theory-and-approach)
+* [Results and Demo](#results-and-demo)
+* [Future Work](#future-work)
+* [Troubleshooting](#troubleshooting)
+* [Contributors](#contributors)
+* [Acknowledgements and Resources](#acknowledgements-and-resources)
+* [License](#license)
+
+---
+
+# <ins>About the Project
+
+**Modeling and Control of Two-Legged Wheeled Robot: In this project, we will be building a two wheeled robot (with a hip and knee joint) which can balance at variable heights including a provision for jumping.**
+
+---
+
+## <ins>Tech Stack
+
+* <font color="red">C</font>
+* <font color="red">Python</font>
+* <font color="red">SolidWorks</font>
+* <font color="red">ESP-IDF</font>
+* <font color="red">AutoCad</font>
+* <font color="red">Creality</font>
+
+## <ins>File Structure
+```
+- myProject/
+             - CMakeLists.txt
+             - sdkconfig
+             - components/ - component1/ - CMakeLists.txt
+                                         - Kconfig
+                                         - src1.c
+                           - component2/ - CMakeLists.txt
+                                         - Kconfig
+                                         - src1.c
+                                         - include/ - component2.h
+             - main/       - CMakeLists.txt
+                           - src1.c
+                           - src2.c
+
+             - build/
+```
+---
+# <ins>Getting Started
+
+## <ins>Prerequisites
+
+1. Used [Solidworks](https://www.solidworks.com/) for Designing our Robot.
+
+2. Used [Creality](https://www.creality.com/pages/download) Software used for 3D Printing
+
+3. Used [ESP-IDF](https://docs.espressif.com/projects/esp-idf/en/latest/esp32/get-started/) for building and flashing the code on ESP-32 Microcontroller.
+
+4. Used [Python](https://www.python.org/) Scripts for calculation of specifications of Bot.
+
+5. Used [Matplotlib](https://matplotlib.org/) for plotting the graphs.
+
+## <ins>Installations
+
+1. Clone the Repository 
+    ```git clone git@github.com:KudoC0nan/Balance_On_Wheels.git```
+2. Install ESP-IDF on your System
+    
+    [For Windows](https://docs.espressif.com/projects/esp-idf/en/latest/esp32/get-started/windows-setup.html#get-started-windows-first-steps)
+    
+    [For Linux and MacOS](https://docs.espressif.com/projects/esp-idf/en/latest/esp32/get-started/linux-macos-setup.html)
+
+## <ins>Execution
+
+1. Open ESP-IDF Terminal on Windows or Terminal on Linux
+
+2. Run get_idf in Linux.
+
+3. Goto The Repository 
+    ```cd Balance_On_Wheels```
+
+4. Goto Self Balancing Code at Minimum Height
+    ```cd Self_Balancing_Servo_set_zero```
+
+5. Run Command for Building the Code
+    ```idf.py build ```
+
+6. Run Command for Opening the Configurations Tab to Setup the WIFI.
+    ```idf.py menuconfig```
+
+7. Run Command on Flashing the Code on our Bot.
+    ```idf.py flash monitor```
+---
+# <ins>Theory And Approach
```

Comment:
You have mentioned all Theory but adding explanation about design, why did you plot the graph would be helpful

Reply:
Minimize the general theory , and try writing in your own words.

Reply:
![Screenshot from 2022-10-05 17-36-25](https://user-images.githubusercontent.com/96683880/194056811-5861fea7-0bb4-4717-85b9-c5afda9d1385.png)

Can we also make a table like this for our design?

Reply:
Added the Table,picture of cad design and made required changes